src/platform/packages/shared/kbn-discover-utils/src/utils/get_message_field_with_fallbacks.ts

```diff
@@ -19,6 +19,9 @@ export const getMessageFieldWithFallbacks = (
     fieldConstants.MESSAGE_FIELD,
     fieldConstants.ERROR_MESSAGE_FIELD,
     fieldConstants.EVENT_ORIGINAL_FIELD,
+    fieldConstants.EXCEPTION_MESSAGE_FIELD,
+    fieldConstants.ERROR_EXCEPTION_MESSAGE,
+    fieldConstants.OTEL_ATTRIBUTES_EXCEPTION_MESSAGE,
   ] as const;
 
   for (const rank of rankingOrder) {
```

Comment:
This looks pretty much the same as implemented in `src/platform/packages/shared/kbn-discover-utils/src/utils/get_log_field_with_fallback.ts`, can you unify it?